src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1242,4 +1239,10 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Make_0_inheritable" xml:space="preserve">
     <value>Make '{0}' inheritable</value>
   </data>
+  <data name="Cast_0_to_1" xml:space="preserve">
+    <value>Cast '{0}' to '{1}'</value>
+  </data>
+  <data name="Add_explicit_cast" xml:space="preserve">
+    <value>Add explicit cast</value>
```

Comment:
why do we have VB specific resources for this?  I would think we woudl be using the same strings for C# and VB (so it should move to Features).